src/database/queries/checkAdmin.js

```diff
@@ -0,0 +1,14 @@
+const dbConnection = require('../db_connection');
+
+const getAdmin = (username, cb) => {
+  const sql = {
+    text: 'SELECT * FROM admin WHERE user_name = $1 ',
+    values: [username],
+  };
+  dbConnection.query(sql, (err, result) => {
+    if (err) return cb(err);
+    return cb(null, result.rows);
```

Comment:
your admin's `user_name` is `UNIQUE` so you are sure that this query will return a single row
so you can send `result.rows[0]`
so you can use it directly in `test` and `controller`

Reply:
@RamyAlshurafa and what if i want to add another admin ?!!